### DISCUSSION TITLE
Why  external images are not loaded in preview mode

### DISCUSSION THREAD

**Discussion by AmirEzati** [directus/directus#13682] [Archive]
I created a link to an external image in one of the markdown fields but it is not shown when I hit the preview button even though the image link is valid:

ex:
`![myImage](ImagePath)`

![Bildschirmfoto 2022-06-02 um 11 32 57](https://user-images.githubusercontent.com/30486048/171601054-3bc06817-2ee6-42b1-8e54-7b3d59e4e14b.png)

**Comment by br41nslug** [comment]
Are you getting any CORS/CSP errors in the browser?

**Reply by AmirEzati** [reply]
yes, that is the issue then.
thanks a lot @br41nslug
